test/core/security/security_connector_test.cc

```diff
@@ -340,6 +341,29 @@ static grpc_ssl_roots_override_result override_roots_permanent_failure(
   return GRPC_SSL_ROOTS_OVERRIDE_FAIL_PERMANENTLY;
 }
 
+static void test_ipv6_address_san(void) {
+  const char* addresses[] = {
+      "2001:db8::1",
+      "fe80::abcd:ef65:4321%em0",
+      "fd11:feed:beef:0:cafe::4",
+  };
+  const char* san_ips[] = {
+      "2001:db8::1",
+      "fe80::abcd:ef65:4321",
+      "fd11:feed:beef:0:cafe::4",
+  };
+  tsi_peer peer;
+  GPR_ASSERT(tsi_construct_peer(1, &peer) == TSI_OK);
+  for (size_t i = 0; i < GPR_ARRAY_SIZE(addresses); i++) {
+    GPR_ASSERT(tsi_construct_string_peer_property_from_cstring(
+                   TSI_X509_SUBJECT_ALTERNATIVE_NAME_PEER_PROPERTY, san_ips[i],
```

Comment:
This line seems not formatted properly. Could you please re-run grpc_clang_format?